docs/tutorial.rst

```diff
@@ -51,13 +51,16 @@ The ``.gitignore`` will remain and keep Git from not tracking the directory.
 Detecting Dates & Versions
 --------------------------
 
-``towncrier`` needs to know what version your project is, and there are two ways you can give it:
+``towncrier`` needs to know what version your project is. These are the ways you can provide it (and their order of precedence):
```

Comment:
I think it's best to use [Semantic newlines](https://sembr.org/) for documentation content.

Maybe also update this page
https://towncrier.readthedocs.io/en/latest/contributing.html

I guess that not every operaition needs the version.
I don't expect to requiere the version for a `create` command.

It's not 100% clear what's the order of precendece.

```suggestion
Most ``towncrier`` commands, like `build` or `check`, needs to know what version your project is.
These are the ways you can provide it and their order of precedence.
A lower precedence number will overwrite any other method with a higher number:
```